core/src/main/java/hudson/Proc.java

```diff
@@ -217,7 +222,13 @@ public LocalProc(String[] cmd,String[] env,InputStream in,OutputStream out, File
         public LocalProc(String[] cmd,String[] env,InputStream in,OutputStream out,OutputStream err,File workDir) throws IOException {
             this( calcName(cmd),
                   stderr(environment(new ProcessBuilder(cmd),env).directory(workDir), err==null || err== SELFPUMP_OUTPUT),
-                  in, out, err );
+                  in, out, err, true );
```

Comment:
Would be preferable to just call the new constructor below to avoid the logic duplication

Reply:
Done